src/FindTranslocations.cpp

```diff
@@ -54,7 +174,7 @@ int main(int argc, char *argv[]) {
 						("coverage",     po::value<float>(), "do not compute coverage from bam file, use the one specified here (must be used in combination with --insert --insert-std)")
 						("insert",       po::value<float>(), "do not compute insert size from bam file, use the one specified here (must be used in combination with --coverage --insert-std)")
 						("insert-std",   po::value<float>(), "do not compute insert size standard deviation from bam file, use the one specified here (must be used in combination with --insert --coverage)")
-						;
+						("extract",       po::value<string>(), "extracts the reads involved in an event, ""CHRA startPosA endPosA CHRB startPosB endPosB"" ");
```

Comment:
Maybe the input should be the output file from FindTranslocation or similar to allow one to extract more than one event